download.html

```diff
@@ -44,30 +44,30 @@ <h3>BSD/Linux distributions</h3>
           <p>xmonad is included in many distributions' package repositories. See if yours is one:</p>
           <div class="distros">
             <ul>
-              <li><a href="http://www.openbsd.org/cgi-bin/cvsweb/ports/x11/xmonad/">OpenBSD</a></li>
-              <li><a href="http://pkgsrc.se/wip/xmonad">NetBSD</a></li>
+              <li><a href="https://www.openbsd.org/cgi-bin/cvsweb/ports/x11/xmonad/">OpenBSD</a></li>
+              <li><a href="https://pkgsrc.se/wip/xmonad">NetBSD</a></li>
               <li><a href="https://www.freshports.org/x11-wm/hs-xmonad/">FreeBSD</a></li>
             </ul>
           </div>
           <div class="distros">
             <ul>
               <li><a href="https://packages.gentoo.org/packages/x11-wm/xmonad">Gentoo</a></li>
-              <li><a href="http://packages.debian.org/xmonad">Debian</a> (<a href="http://people.debian.org/~jps/etch">etch</a>) </li>
-              <li><a href="http://packages.ubuntu.com/search?keywords=xmonad">Ubuntu</a></li>
+              <li><a href="https://packages.debian.org/xmonad">Debian</a> (<a href="https://people.debian.org/~jps/etch">etch</a>) </li>
```

Comment:
```suggestion
              <li><a href="https://packages.debian.org/xmonad">Debian</a></li>
```

Not sure if we can remove this "etch" link.

Reply:
I think it's fine.